dev/devServer.js

```diff
@@ -0,0 +1,49 @@
+const esbuild = require("esbuild");
+const chokidar = require("chokidar");
+const liveServer = require("live-server");
+const { esm } = require("../buildConfigs");
+
+(async () => {
+  // Builder for the react-search package
+  const pkgBuilder = await esbuild.context(esm);
+
+  // Builder for the development page
+  const devPageBuilder = await esbuild.context({
+    bundle: true,
+    define: {
+      "process.env.NODE_ENV": JSON.stringify(
+        process.env.NODE_ENV || "development"
+      ),
+    },
+    entryPoints: ["dev/index.tsx"],
+    minify: process.env.NODE_ENV === "production",
```

Comment:
Do we need this option? Seems like we'd never want to minify this for local dev?

Reply:
https://github.com/vectara/react-search/pull/4/commits/ad9a0b57138d8f7c6269c15dc1e42377b95eae11